### PR TITLE
Allows to specify the list of tasks performed by the bot from config env

### PR DESCRIPTION
**Pull request by lmignon**
ping @sbidoul 

* [x] tests

### CODE REVIEW

File: src/oca_github_bot/config.py [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+def switchable(switch_name=None):
+    def wrap(func):
+        @wraps(func)
+        def func_wrapper(*args, **kwargs):
+            sname = switch_name
+            if switch_name is None:
+                sname = func.__name__
+
+            if BOT_TASKS != ["all"] and sname not in BOT_TASKS:
+                _logger.debug("Method%s skipped (Disabled by config)", sname)
```
Comment (sbidoul) [inline src/oca_github_bot/config.py]:
```suggestion
                _logger.debug("Method %s skipped (Disabled by config)", sname)
```

File: environment.sample [hunk 2]
```diff
@@ -28,3 +28,10 @@ ODOO_LOGIN=
 ODOO_PASSWORD=
 
 #SENTRY_DSN=
+
+# Coma separated list of task to run
+# By default all configured tasks are run.
+# Available tasks:
+#  delete_branch,tag_approved,tag_ready_to_merge,gen_addons_table,
+#  gen_addons_readme,gen_addons_icon,setuptools_odoo,merge_bot
+# BOT_TASKS = ["all"] (default)
```
Comment (sbidoul) [inline environment.sample]:
```suggestion
#BOT_TASKS=all
```

File: src/oca_github_bot/config.py [hunk 3]
```diff
@@ -23,3 +46,10 @@
 SENTRY_DSN = os.environ.get("SENTRY_DSN")
 
 DRY_RUN = os.environ.get("DRY_RUN", "").lower() in ("1", "true", "yes")
+
+# coma separated list of task to run
+# by default all configured tasks are run.
+# defined tasks:
+#  delete_branch,tag_approved,tag_ready_to_merge,gen_addons_table,gen_addons_readme,
+#  gen_addons_icon,setuptools_odoo,merge_bot
+BOT_TASKS = os.environ.get("TASKS", "all").split(",")
```
Comment (sbidoul) [inline src/oca_github_bot/config.py]:
```suggestion
BOT_TASKS = os.environ.get("BOT_TASKS", "all").split(",")
```

**Comment by lmignon** [comment]
Changes done. Thank you for the review @sbidoul

**Pull request merged**